packages/grafana-ui/src/components/Card/Card.story.tsx

```diff
@@ -133,9 +132,6 @@ export const Full: Story<Props> = ({ disabled }) => {
       <Card.Figure>
         <img src={logo} alt="Prometheus Logo" />
       </Card.Figure>
-      <Card.Tags>
-        <TagList tags={['firing', 'active', 'test', 'testdata', 'prometheus']} onClick={action('Clicked tag')} />
-      </Card.Tags>
```

Comment:
TagList doesn't work with full card layout and we don't have any components that use full card layout atm. In case we'll need to display tags not on the right side of the card, the Tags components will need to be modified to accept a position property.